cmake/onnxruntime_providers.cmake

```diff
@@ -689,10 +689,11 @@ if (onnxruntime_USE_TENSORRT)
       target_compile_options(nvonnxparser_static PRIVATE /FIio.h /wd4100)
       target_compile_options(nvonnxparser PRIVATE /FIio.h /wd4100)
     endif()
-    include_directories(${TENSORRT_INCLUDE_DIR})
```

Comment:
can we consolidate line 713 and 716 then?  seems like with this change we no longer need TENSORRT_INCLUDE_DIR in line 713
which means it will be the same as line 716.  and we can consolidate and move out of the if() else() endif() after line 717 ?

Reply:
Done